include/bluetooth/services/fast_pair.h

```diff
@@ -30,18 +37,76 @@ enum bt_fast_pair_adv_mode {
 
 	/** Fast Pair not discoverable advertising, hide UI indication. */
 	BT_FAST_PAIR_ADV_MODE_NOT_DISCOVERABLE_HIDE_UI_IND,
+
 	/** Number of Fast Pair advertising modes. */
 	BT_FAST_PAIR_ADV_MODE_COUNT
 };
 
+/** @brief Fast Pair advertising battery mode. Used to generate advertising packet.
+ *
+ * Battery data can be included in advertising packet only if the Fast Pair Provider is in Fast Pair
+ * not discoverable advertising mode. To prevent tracking, the Fast Pair Provider should not include
+ * battery data in the advertising packet all the time.
+ */
+enum bt_fast_pair_adv_battery_mode {
+	/** Do not advertise battery data. */
+	BT_FAST_PAIR_ADV_BATTERY_MODE_NONE,
+
+	/** Show battery data UI indication. */
+	BT_FAST_PAIR_ADV_BATTERY_MODE_SHOW_UI_IND,
+
+	/** Hide battery data UI indication. */
+	BT_FAST_PAIR_ADV_BATTERY_MODE_HIDE_UI_IND,
+
+	/** Number of Fast Pair advertising battery modes. */
+	BT_FAST_PAIR_ADV_BATTERY_MODE_COUNT
+};
+
+/** @brief Index of Fast Pair battery component. */
+enum bt_fast_pair_battery_comp {
```

Comment:
Maybe `bt_fast_pair_component`? The enum denotes given component, it may not only be related to battery